### ISSUE TITLE
Problem with hovering event on timeseries bar chart (bb-event-rect not set properly?)

### ISSUE THREAD

**Issue by ugoQ** [naver/billboard.js#2913]
Hi there,
I have a problem on a bar chart where the tooltip and event don't work on the edges on the chart (on a part of the first and last bar)

Here you can see if I hover on the end or the start of the chart, the tooltip is not shown until the cursor is "far" in the bar:
![billboardHover1](https://user-images.githubusercontent.com/50490703/197568395-4c99aa43-d647-4145-9eb5-fa1c7703b9cd.gif)

It gets worse the more bars (columns) I have on the same chart (don't mind the blue color everywhere it's just for the example):
![billboardHover2](https://user-images.githubusercontent.com/50490703/197568434-69099344-419d-43ea-abc2-dc1fcbc937bb.gif)

My problem is it doesn't show the tooltip and the `onclick` event attached to each bar doesn't work either (I use the non grouped tooltips but it seems it won't work on grouped tooltips either, see below the steps to reproduce).

The `rect.bb-event-rect` seems to have the good size though.
![image](https://user-images.githubusercontent.com/50490703/197568330-6947ebc6-f270-41d3-958f-a27192215e02.png)

If you want to reproduce easily, just go there https://naver.github.io/billboard.js/demo/#Chart.TimeseriesChart

And use this (with `tooltip: {grouped: false}` or not, you will see the same thing):
```
var chart = bb.generate({
  data: {
    x: "x",
    columns: [
	["x", "2013-01-01", "2013-01-02", "2013-01-03", "2013-01-04", "2013-01-05", "2013-01-06"],
	["data1", 30, 200, 100, 400, 150, 250],
	["data2", 130, 340, 200, 500, 250, 350],
	["data3", 400, 500, 450, 700, 600, 500],
	["data4", 400, 500, 450, 700, 600, 500]

    ],
    type: "bar", // for ESM specify as: line()
  },
  axis: {
    x: {
      type: "timeseries",
      tick: {
        format: "%Y-%m-%d"
      }
    }
  },
  bindto: "#timeseriesChart"
});
```

I saw the same behavior on Chrome and Firefox.
Thanks in advance for any help!

**Comment by netil** [comment]
Hi @ugoQ, thanks for the report.
The issue was fixed and released the patch `3.6.2`.

**Comment by ugoQ** [comment]
Just tried it on the `3.6.2`, works like a charm.

Man I'm so thankful for this fix, that's not the first time I open an issue on this repo and every time you deliver!
Really appreciate your work, thanks!
